05/style/desktop.css

```diff
@@ -0,0 +1,73 @@
+.green {
+    border: 3px solid green;
+    margin-bottom: 5px;
+    padding: 5px;
+}
+
+.blue {
+    border: 3px solid blue;
+    max-width: 1000px;
+    margin: 0 auto;
+}
+
+.orange {
+    background-color: orange;
+    width: 100%;
+    height: 200px;
+    margin-bottom: 10px;
+}
+
+footer .orange {
+    width: 200px;
+    height: 50px;
+    margin: 10px auto;
+}
+
+header > .blue {
```

Comment:
Jeśli chcemy robić zgodnie z BEM to lepiej zrobić modyfikator np. `.blue--header` zamiast tworzyć "rozbudowane" selektory :)